core/src/test/java/com/linecorp/armeria/server/HttpServerHandlerTest.java

```diff
@@ -103,6 +104,6 @@ void httpResponseExceptionIsNotLoggedAsRequestCause() {
         await().untilAsserted(() -> {
             assertThat(logHolder.get().requestHeaders().path()).isEqualTo("/httpResponseException");
         });
-        assertThat(logHolder.get().requestCause()).isNull();
+        assertThat(logHolder.get().requestCause()).isInstanceOf(AbortedStreamException.class);
```

Comment:
The request is not subscribed but aborted in [`HttpServerHandler`](https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java#L416) so the cause is changed.
Should I change not to log this exception when the request is not subscribed? 🤔

Reply:
Hmm. Most `GET` request handler will not subscribe at all. Perhaps we should handle the case where it is certain that request (or response) body will be empty?

Reply:
However, such logic isn't part of the `StreamMessage` implementations. Could probably be done in the places that call `abort()`?

Reply:
> Perhaps we should handle the case where it is certain that request (or response) body will be empty?

That's a nice idea. `EmptyFixedStreamMessage` works in the same way. Let me make a change for that. Thanks!